docs/en/observability/monitor-k8s/monitor-k8s-add-integration.asciidoc

```diff
@@ -0,0 +1,325 @@
+[discrete]
+[[monitor-kubernetes-integration]]
+== Part 1: Add and Configure the Kubernetes integration
+
+To start collecting logs and metrics from your Kubernetes clusters, first add the {integrations-docs}/kubernetes[Kubernetes integration] to your deployment and configure the metrics and logs you want to collect.
+
+[discrete]
+[[monitor-k8s-add-integration]]
+=== Step 1: Add the Kubernetes integration to your deployment
+
+Follow these steps to add the Kubernetes integration to your deployment:
+//add screenshots if necessary
+
+. Select *Add integrations* from your deployment's homepage.
+. Enter "Kubernetes" in the search bar, and select the *Kubernetes* integration.
+. Click *Add Kubernetes* at the top of the Kubernetes integration page.
+. Click *Add integration only (skip agent installation)* at the bottom of the Add integration page.
+
+You're now on the *Add Kubernetes integration* page.
+From here, you'll configure the integration to collect metrics and logs from the different Kubernetes components.
+Continue to <<monitor-k8s-configure-integration, Step 2>> to configure your integration.
+
+[discrete]
+[[monitor-k8s-configure-integration]]
+=== Step 2: Configure your Kubernetes integration
+
+The Kubernetes integration can fetch metrics and logs from several components.
+From the *Add Kubernetes integration* page, configure the options for the metrics and logs you want to collect.
+The following sections outline the configuration options.
+
+*Collect Metrics*
+
+Collecting metrics about Kubernetes clusters and the workloads running on top of them is a key aspect of Kubernetes observability.
+However, collecting metrics from Kubernetes poses some challenges.
+You need to collect metrics about the resources running on "physical" machines as well as the containers and pods.
+Specifically, you need to monitor the health and performance of:
+
+* The hosts where Kubernetes components are running. Each host produces metrics
+like CPU, memory, disk utilization, and disk and network I/O.
+
+* Kubernetes containers, which produce their own set of metrics.
+
+* The applications running as Kubernetes pods, such as application servers and
+databases, each producing its own set of metrics.
+
+Instead of using multiple technologies to collect metrics, deploy the {agent} along with the Kubernetes integration to monitor all layers of your technology stack.
```

Comment:
```suggestion
 {agent} along with the Kubernetes integration provides a unified solution to monitor all layers of your {k8s} technology stack (Instead of using multiple technologies to collect metrics)
```